plugins/tekton/README.md

```diff
@@ -1,89 +1,114 @@
-4. The `ClusterRole` must be granted for custom resources (pipelineruns and taskruns) to `ServiceAccount` accessing the cluster. If you have the Backstage Kubernetes Plugin configured, the [ClusterRole](https://backstage.io/docs/features/kubernetes/configuration#role-based-access-control) is already granted.
-   You can use the following code to grant the `ClusterRole` for custom resources:
-
-   ```yaml
-   apiVersion: rbac.authorization.k8s.io/v1
-   kind: ClusterRole
-   metadata:
-     name: backstage-read-only
-   rules:
-     # ...
-     - apiGroups:
-         - tekton.dev
-       resources:
-         - pipelineruns
-         - taskruns
-       verbs:
-         - get
-         - list
-   ```
+  You can use the following code to grant the `ClusterRole` for custom resources:
 
-5. To get the resources from a Kubernetes cluster, add the following annotations to the entity's `catalog-info.yaml`:
+  ```yaml
+  ...
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: backstage-read-only
+    rules:
+      ...
+      - apiGroups:
+          - tekton.dev
+        resources:
+          - pipelineruns
+          - taskruns
+        verbs:
+          - get
+          - list
 
-   ```yaml title="catalog-info.yaml"
-   annotations:
-     backstage.io/kubernetes-id: <BACKSTAGE_ENTITY_NAME>
-     backstage.io/kubernetes-namespace: <RESOURCE_NS>
-   ```
+  ```
 
-   The Kubernetes plugin identifies if the provided entity contains Kubernetes resources and from which namespace the plugin receives the resources based on the previous annotations.
+- The following annotation is added to the entity's `catalog-info.yaml` file to identify whether an entitiy contains the Kubernetes resources:
 
-6. Add the following annotation to entity's `catalog-info.yaml` file to view the latest `PipelineRun` visualization in the CI/CD tab of the application:
+  ```yaml
+  annotations:
+  backstage.io/kubernetes-id: <BACKSTAGE_ENTITY_NAME>
+  ```
 
-   ```yaml title="catalog-info.yaml"
-   annotations:
-     # ...
-     janus-idp.io/tekton-enabled: 'true'
-   ```
+  You can also add the `backstage.io/kubernetes-namespace` annotation to identify the Kubernetes resources using the defined namespace.
 
-7. You can also add a custom label selector, which Backstage uses to find the resources. The label selector takes precedence over the ID annotation.
-   `'backstage.io/kubernetes-label-selector': 'app=my-app,component=front-end`
-8. You must add the following label to the resources so that the Kubernetes plugin gets the Kubernetes resources from the requested entity:
+  ```yaml
+  annotations:
+    backstage.io/kubernetes-namespace: <RESOURCE_NS>
+  ```
 
-   ```yaml title="catalog-info.yaml"
-   'backstage.io/kubernetes-id': <BACKSTAGE_ENTITY_NAME>`
-   ```
+- The following annotation is added to the `catalog-info.yaml` file of entity to view the latest `PipelineRun` in the CI/CD tab of the application:
```

Comment:
Again - at what time is the annotation added? Or do you want to instruct the user to ad this annotation? Also "file of *the* entity"